tetris.py

```diff
@@ -222,8 +222,15 @@ def block_rotator(self):
         Rotates the current block
         """
         self.rotate_counter += 1
+        if any(x <= -1 or x >= game_width for (x, y) in self.get_current_block()) or any(
+            (x, y) in self.landed_blocks for x, y in self.get_current_block()
+        ):
```

Comment:
```suggestion
        if any(x not in range(game_width) or (x, y) in self.landed_blocks for (x, y) in self.get_current_block()):
```

Reply:
Would you mind me keeping the above code commented out?

The last one is nice but it's something I wouldn't come up with at the moment :)

Reply:
We can keep it as is. It already works.